frontend/packages/local-storage-operator-plugin/src/components/local-volume-set/create-local-volume-set.scss

```diff
@@ -37,3 +37,14 @@
 .lso-create-lvs__disk-size-form-group-max-input {
   max-width: 100px;
 }
+
+// Restrict node-list widths
+.lso-create-lvs__node-list {
+  @media (min-width: 769px) {
+    max-width: 100% !important;
+  }
+
+  @media (min-width: 1200px) {
+    max-width: 70% !important;
+  }
```

Comment:
Instead of defining the same styles at two places, can we define at once and use at both?